include/revng/Yield/Internal/Instruction.h

```diff
@@ -0,0 +1,79 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include <limits>
+#include <string>
+
+#include "revng/ADT/SortedVector.h"
+#include "revng/Model/VerifyHelper.h"
+#include "revng/Support/MetaAddress.h"
+#include "revng/Support/MetaAddress/YAMLTraits.h"
+#include "revng/Yield/Internal/TagType.h"
+
+namespace yield {
+using ByteContainer = llvm::SmallVector<uint8_t, 16>;
+}
+
+/* TUPLE-TREE-YAML
+
+name: Instruction
+type: struct
+fields:
+  - name: Address
+    type: MetaAddress
+  - name: Bytes
+    type: yield::ByteContainer
+  - name: Raw
+    type: std::string
+
+  - name: Tags
+    sequence:
+      type: std::vector
+      elementType: yield::Tag
+    optional: true
+
+  - name: Opcode
+    type: std::string
+    optional: true
+  - name: Comment
+    type: std::string
+    optional: true
+  - name: Error
+    type: std::string
+    optional: true
```

Comment:
What is this for? Doc.

Reply:
Sure, sorry.

Reply:
[Done](https://github.com/Cvelth/revng/blob/snapshot/wip_cfg_in_svg/include/revng/Yield/Instruction.h#L50).